windows/src/developer/TIKE/dialogs/UfrmOptions.dfm

```diff
@@ -417,4 +441,11 @@ inherited frmOptions: TfrmOptions
     Left = 212
     Top = 403
   end
+  object dlgBrowseDefaultProjectPath: TBrowse4Folder
+    InitialDir = 'C:\Users\mcdurdin\Desktop\'
```

Comment:
Is this used? If so, is there a registry setting to get the Desktop path?

Reply:
It's not used, but I will remove it from the .dfm.